ci/replace_variables_in_path.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+set -uo pipefail
+
+BEFORE=$(echo "$2" | sed -r "s/include::(.*)\{(.*)\}(.*)\[\]/\1/")
+VAR=$(echo "$2" | sed -r "s/include::(.*)\{(.*)\}(.*)\[\]/\2/")
+AFTER=$(echo "$2" | sed -r "s/include::(.*)\{(.*)\}(.*)\[\]/\3/")
```

Comment:
These three are the same regular expression. Let's make it obvious by aligning them
```suggestion
BEFORE=$(echo "$2" | sed -r "s/include::(.*)\{(.*)\}(.*)\[\]/\1/")
VAR=$(echo "$2"    | sed -r "s/include::(.*)\{(.*)\}(.*)\[\]/\2/")
AFTER=$(echo "$2"  | sed -r "s/include::(.*)\{(.*)\}(.*)\[\]/\3/")
```

Reply:
done